tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php

```diff
@@ -436,249 +392,9 @@ interface Test
         /**/TestInterface65    {}
             ',
         ];
-    }
 
-    public static function provideFixingTraitsCases(): iterable
-    {
-        return self::provideClassyCases('trait');
-    }
-
-    /**
-     * @param array<string, mixed> $expected
-     *
-     * @dataProvider provideClassyDefinitionInfoCases
-     */
-    public function testClassyDefinitionInfo(string $source, array $expected): void
-    {
-        Tokens::clearCache();
-        $tokens = Tokens::fromCode($source);
-
-        $method = new \ReflectionMethod($this->fixer, 'getClassyDefinitionInfo');
```

Comment:
this test got removed? similar for few others

Reply:
Do we want to test private methods? Aren't testing the fixes normally enough?

Reply:
i would not mix in the single PR refatoring of test case organisation and removal of testcases/test methods. make it hard to review and extending decision process.

if I wouldn't notice, I would be blindly sure that we only dropped the `if (PHP_VERSION >...)` conditions

Reply:
Well, this PR is not about dropping `if (PHP_VERSION >...)` conditions, it's about unifying tests for fixer, the only relation is that tests with such conditions are selected first to get remove https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7580/files#diff-cffacea4c090548d048cfc3ba0829a6d525df7f6f190c07327e395b429b8dcc6R357 as soon as possible.

Do we want to have more method names allowed in fixer's tests? If we want such hacky tests maybe at least we can move them to other classes and add `@coversNothing` not to falsly show coverage?

Reply:
I would suggest to have one PR for thing like PHP_VERSION>, one for naming, etc, if there would be more unification and tests behind it. (if not really and PR is full, also perfect!)

yet , i would suggest to not removed existing tests. They were added for some reason and I'm out of capacity to determine why and if still needed, while there is no harm to keep them longer in codebase

Reply:
How would you like to prevent them? In this PR unification of the test class means all the public methods must be some fixed list - do you want to allow `testExtra` method or something like that?

Reply:
What do you mean "prevent them"? I do not want to prevent existing tests to exist.
I care more about having the existing test not removed than about having a test method naming unified.

for now, I'm OK to have the test case recovered "as-is", the way we have it present on master, and any realignment to happen outside of this massive PR, to not block merge of this PR (imagine if I would run other PR that is doing some massive cleanup on other aspect and we have gazillion of conflicts)

Reply:
I mean "preserve" :) I've reverted them for now.

Reply:
thanks!
let's handle them as separate discussion in q1